gramhopper/triggers/text_triggers.py

```diff
@@ -42,13 +42,14 @@ def __init__(self, substring: Union[str, List[str]], exact: bool = False):
     def validate_strings_list(strings):
         return all(map(lambda element: isinstance(element, str), strings))
 
+
 class _HasExactWordTrigger(_HasSubstringTrigger):
     def __init__(self, word: Union[str, List[str]]):
         try:
             super().__init__(word, exact=True)
         except TypeError:
             raise TypeError(f'Parameter \'word\' should be either a string or a list of strings '
-                            '({type(substring)} given)')
+                            f'({type(word)} given)')
```

Comment:
Is this a fix? I don't quite follow.

Reply:
Yes. The previous version just wouldn't format the inner variable (which I left `substring` instead of `word` by mistake).